collectd-docker.py

```diff
@@ -365,6 +365,9 @@ def build_network_stats_for(stats):
         for key, value in interface_stats.iteritems():
             network_stats[key] += value
 
+    # Remove non-aggregated network stats
+    del stats['networks']
```

Comment:
This function mutates a `stats` dictionary that's passed through the flow of the program. We store the aggregated network stats under the `network` key, and want to discard the original `networks` key, otherwise it will be submitted to Librato and the customer will be charged for the additional metrics.